tests/Rules/CallMethodRuleTest.php

```diff
@@ -14,10 +17,10 @@ class CallMethodRuleTest extends RuleTestCase
     protected function getRule(): Rule
     {
         $broker = $this->createBroker();
-        $ruleLevelHelper = new RuleLevelHelper($broker, true, true, true);
+        $ruleLevelHelper = new RuleLevelHelper($broker, true, true, true, false);
         return new CallMethodsRule(
             $broker,
-            new FunctionCallParametersCheck($ruleLevelHelper, true, true, true, true),
+            new FunctionCallParametersCheck($ruleLevelHelper, new NullsafeCheck(), new PhpVersion(PHP_VERSION_ID), new UnresolvableTypeHelper(), true, false, false, false),
```

Comment:
especially I am not sure about these 3 new added `false` passed for the 3 new args:
![grafik](https://user-images.githubusercontent.com/120441/139667621-ba8f1428-8338-41ef-b136-2acebe5299bc.png)

Reply:
I'm just looking around, but from looking into phpstan core, I don't see that the booleans are added to the FunctionCallParametersCheck, only the last boolean was removed.

Frankly, it seems odd that the instantiation of FunctionCallParametersCheck did not include the nullsafe check, phpversion and helper before and was just passing the booleans instead..

Feels like it should either stay the same as:
`...true, true, true, true` unless there's a reason for disabling the checks..

Anyways, I temporarily updated phpstan-safe-rule in my project with this fork and it seems it works just as well as previously. Hope this helps..